Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -315,12 +406,6 @@ SWIFT_CLASS_NAMED("CustomerInfo")
 /// note:
 /// This can be nil, see -<code>Purchases.restorePurchases(completion:)</code>
 @property (nonatomic, readonly, copy) NSString * _Nullable originalApplicationVersion;
-/// The underlying data for this <code>CustomerInfo</code>.
-/// note:
-/// the content and format of this data isn’t documented and is subject to change.
-/// it’s only meant for debugging purposes or for getting access to future data
-/// without updating the SDK.
-@property (nonatomic, readonly, copy) NSDictionary<NSString *, id> * _Nonnull rawData;
```

Comment:
Why is this gone too? So weird.

Reply:
Looking into this.

Reply:
I'm so confused about this. The property _is_ there (and tested in `APITester`s since #1550).
Using it works FWIW.

What changed since the last release is that this is an extension now:
```swift

extension CustomerInfo: RawDataContainer {
    // Docs inherited from protocol
    // swiftlint:disable missing_docs
    public var rawData: [String: Any] {
        return self.data.response.rawData
    }
}
```

Reply:
looks like we're missing `@objc`?

Reply:
Ooooooh that's what it is!